internal/rbd/nodeserver.go

```diff
@@ -241,8 +241,13 @@ func populateRbdVol(
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
-	if rv.Mounter == rbdDefaultMounter &&
-		!isKrbdFeatureSupported(ctx, strings.Join(rv.ImageFeatureSet.Names(), ",")) {
+	isFeatureExist, err := isKrbdFeatureSupported(ctx, strings.Join(rv.ImageFeatureSet.Names(), ","))
+	if errors.Is(err, os.ErrNotExist) {
+		// do not fallback if /sys/bus/rbd/supported_features is absent and we are
+		// not in a position to prepare krbd feature attributes
+		isFeatureExist = true
```

Comment:
should this default to `true`? I would expect that if the kernel is too old, and the file does not exist, it is assumed the feature is not available either.

Reply:
@nixpanic are you saying, If the kernel is too old and the file doesn't exist and hence we set krbdFeatures to 0, so fall back to rbd-nbd?

Reply:
Still waiting for a reply on this, thanks!

Reply:
@nixpanic 
Can you check now! Thanks.